go/vt/vttablet/onlineddl/executor.go

```diff
@@ -627,18 +627,8 @@ func (e *Executor) terminateVReplMigration(ctx context.Context, uuid string) err
 		// silently skip error; stopping the stream is just a graceful act; later deleting it is more important
 		_, _ = tmClient.VReplicationExec(ctx, tablet.Tablet, query)
 	}
```

Comment:
nit: a style comment. The extra braces around this code block are not required.

Reply:
fixed styling